layers/+email/mu4e/packages.el

```diff
@@ -39,9 +39,28 @@
                 'ivy-completing-read
               'helm-completing-read))
 
+      (when mu4e-async
+        (require 'smtpmail-async)
+        (mu4e/set-send-function 'async-smtpmail-send-it))
+
+      (when (fboundp 'imagemagick-register-types)
+        (imagemagick-register-types))
+
+      (let ((dir "~/Downloads"))
```

Comment:
Can we make this configurable?  Or is there some other, more appropriate way of setting this if I want my attachments somewhere else?